enos/modules/get_local_metadata/scripts/build_date.sh

```diff
@@ -1,10 +1,6 @@
 #!/bin/env bash
 set -eu -o pipefail
 
-# Set up the environment for building Vault.
-root_dir="$(git rev-parse --show-toplevel)"
-
-pushd "$root_dir" > /dev/null
-
-IFS="-" read -r VAULT_BUILD_DATE _other <<< "$(make build-date)"
-echo $VAULT_BUILD_DATE
+pushd "$(git rev-parse --show-toplevel)" > /dev/null
+make crt-get-date
```

Comment:
since we are now adding the support for this in `Makefile` we probably can remove these scrips and just call `enos_local_exec` with `inline` commands ?  The default location in CI would be root of the repo.

Reply:
not always, you can use --chdir with Enos. This is safer because it'll always work.